.changeset/flat-squids-search.md

```diff
@@ -0,0 +1,5 @@
+---
+'@makeswift/runtime': patch
+---
+
+Fix radio buttons not rendered on a single select column.
```

Comment:
Can you be more descriptive for nee Makeswift users and specify this is for the built-in Form component?

Reply:
Cool, updated!